event.go

```diff
@@ -75,6 +78,14 @@ func (e *Event) Msg(msg string) {
 	if e == nil {
 		return
 	}
+	if len(e.h) > 0 {
+		e.h[0].Run(e, e.level, e.hasLevel, msg)
+		if len(e.h) > 1 {
+			for _, hook := range e.h[1:] {
+				hook.Run(e, e.level, e.hasLevel, msg)
+			}
+		}
+	}
```

Comment:
Could you factorize this so you don't repeat the same code in `Msgf`?

Reply:
Event.Msgf